### PR TITLE
Add calculation functionality

### PR DESCRIPTION
**Pull request by piero-vic**
# Add calculation functionality
- Add `calculate.js` and `operate.js`.
- Create Button component.
- Add functionality for calculations.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (bimbolabuari) [inline src/components/Button.js:4]: Kindly remove this disabled linter, linting improves readability, removes errors. 👍

Please check for other occurrences 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @piero-vic, 

You have done a good job so far 🚀  

## Highlights 🎉
- Copied the files calculate.js and operate.js into a logic/ directory in your project. ✊
- Imported the files in your Calculator component. 🚀
- Implemented the event handlers. 🎉
- Used state and props 👍

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/components/Button.js (hunk 1 above)
Comment (piero-vic) [reply src/components/Button.js:4]: Hi. One of the requirements for the previous section was to implement all components as class-based. Removing the linters will impede me to to use class-based components. Should I just change them?

**Comment by od-c0d3r** [comment]
Asslam Alaikum @piero-vic  ,😊

Hi👋🏽, Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**